### DISCUSSION TITLE
XML to JSON in Rego

### DISCUSSION THREAD

**Discussion by grubis** [open-policy-agent/community#158] [OPA and Rego]
Can anyone share a clever method to convert an XML string to a JSON object in Rego? I'm using http GET and the result from this particular web service is XML.

**Comment by anderseknert** [comment]
There's currently no such built-in function in Rego. You'd have to either do it outside of policy evaluation—either as part of building the `input`, or have a service in front of the XML API to convert for you—or provide a custom built-in function for that purpose.
